test/translator/common_test.cc

```diff
@@ -34,4 +36,30 @@ TEST(Common, ShouldCorrectlyCallback) {
   translator::DebugLog(buf, 123);
 }
 
+TEST(Common, ShouldNotMemcpy) {
+  scsi_defs::Read6Command cmd;
+  uint8_t buffer[1];
+
+  auto span = absl::MakeSpan(buffer, sizeof(scsi_defs::Read6Command) + 1);
```

Comment:
I'm uncomfortable constructing spans like this that address invalid memory. Just make `buffer` big enough, maybe?